roles/openshift_node_group/tasks/create_config.yml

```diff
@@ -44,6 +44,22 @@
     when: openshift_node_group_edits|length > 0
     run_once: true
 
+  - name: create volume config template
+    template:
+      src: volume-config.yaml.j2
+      dest: "{{ mktempout.stdout }}/volume-config.yaml"
+    when:
+    - configout.results.results.0 == {} and openshift_node_group_name != "" and openshift_node_local_quota_per_fsgroup is defined and openshift_node_local_quota_per_fsgroup != ""
+    run_once: true
```

Comment:
run_once means that it will run only on the first node of a set of nodes. We'd want this file to get deployed to all relevant nodes, right?

Reply:
Nevermind, I should've read the entire PR before commenting.